### ISSUE TITLE
Build failure on 32bit systems

### ISSUE THREAD

**Issue by gibmat** [lestrrat-go/jwx#1155]
**Describe the bug**

v1.2.29 of this library was recently packaged for Debian, and there are build failures on 32bit systems (full log [available here](https://ci.debian.net/packages/g/golang-github-lestrrat-go-jwx/testing/i386/49670803/)). (v1 of the library is required by another package that has not yet updated to v2.)

The following patch fixes the issue; it applies to both the v1 and v2 (with slight offset) branches. I am not sure what your policy is for updating the v1 branch, so I'm opening this issue rather than starting with a pull request.

```
diff --git a/jwk/jwk_test.go b/jwk/jwk_test.go
index b346d49..9cdd309 100644
--- a/jwk/jwk_test.go
+++ b/jwk/jwk_test.go
@@ -1447,7 +1447,7 @@ c4wOvhbalcX0FqTM3mXCgMFRbibquhwdxbU=
 
 type typedField struct {
 	Foo string
-	Bar int
+	Bar int64
 }
 
 func TestTypedFields(t *testing.T) {
diff --git a/jwt/jwt_test.go b/jwt/jwt_test.go
index 24aa375..37512bc 100644
--- a/jwt/jwt_test.go
+++ b/jwt/jwt_test.go
@@ -1116,7 +1116,7 @@ func TestGH375(t *testing.T) {
 
 type Claim struct {
 	Foo string
-	Bar int
+	Bar int64
 }
 
 func TestJWTParseWithTypedClaim(t *testing.T) {
```

**Comment by lestrrat** [comment]
@gibmat  Thank  you, I'll fix.

Does this require a real release? If so, are you in a hurry?
I'm asking because 1. I usually like to let my changes sit for a few days before releasing, and 2. there are a couple of other fixes that I might want to bundle, and so I'm trying to gauge the timing to make a release.

**Comment by gibmat** [comment]
Nope, no rush on my end!

**Comment by lestrrat** [comment]
@gibmat released v1.2.30 and v2.1.1
